src/api/utils.js

```diff
@@ -186,8 +186,8 @@ export function useWebSocket({ kind: _ }) {
   }, [queryClient, webSocket]);
 }
 
-export function useCollection(kind, api, params) {
-  const query = useQuery([kind, params], () => api(params));
+export function useCollection(kind, api, params, queryConfig) {
+  const query = useQuery([kind, params], () => api(params, queryConfig));
```

Comment:
```suggestion
  const query = useQuery([kind, params], () => api(params), queryConfig);
```

`queryConfig` should be passed as the third param to `useQuery`